include/experimental/__p1673_bits/linalg_execpolicy_mapper.hpp

```diff
@@ -12,11 +12,12 @@ inline namespace __p1673_version_0 {
 namespace linalg {
 namespace impl {
 // the execution policy used for default serial inline implementations
-struct inline_exec_t {};
+using inline_exec_t = std::execution::sequenced_policy;
```

Comment:
Please see note above about the "inline" executor.  

Any Standard Algorithm overload that takes an `ExecutionPolicy` template parameter is a "parallel algorithm" (see [[algorithms.parallel] 2](https://eel.is/c++draft/algorithms.parallel#defns-2)).  That changes what the algorithm assumes about the behavior of element access functions. 

Therefore, it's really important that the "inline" executor not use any parallel algorithm, even if `ExecutionPolicy` is `sequenced_policy`.